tools/saqc/saqc.xml

```diff
@@ -233,74 +231,56 @@
-                <validator type="empty_field"/>
+              <param argument="min_periods" type="integer" label="min_periods" help="Minimum number of periods a search window has to contain, for the result of the changepoint detection to be considered valid" min="0"/>
+              <param argument="tolerance" type="text" optional="true" label="tolerance" help="If an offset string is passed, a data chunk of length `offset` right from the start and right before the end of any regime is ignored when calculating a regimes mean for data correcture. This is to account for the unrelyability of data near the changepoints of regimes (Pandas frequency/offset string, e.g., '1D', '2H30M', 'min', 'W-MON')">
+                <validator type="regex" message="Must be a valid Pandas offset/frequency string (e.g., '1D', '2H30M', 'min', 'W-MON')."><![CDATA[^(\s*(\d+(\.\d+)?)?\s*[A-Za-z]+(?:-[A-Za-z]{3})?\s*)+$]]></validator>
               </param>
-              <param argument="min_periods" type="integer" value="" label="min_periods" help="Minimum number of periods a search window has to contain, for the result of the changepoint detection to be considered valid"/>
-              <param argument="tolerance" type="text" optional="true" label="tolerance" help="If an offset string is passed, a data chunk of length `offset` right from the start and right before the end of any regime is ignored when calculating a regimes mean for data correcture. This is to account for the unrelyability of data near the changepoints of regimes"/>
             </when>
             <when value="correctRegimeAnomaly">
-              <param argument="field" type="text" value="" label="Field" help="The name of the variable to process.">
+              <param argument="field" type="data_column" label="field" help="Variable to process" data_ref="data" display="checkboxes" multiple="true"/>
+              <param argument="cluster_field" type="text" label="cluster_field" help="A string denoting the field in data, holding the cluster label for the data you want to correct">
                 <validator type="empty_field"/>
               </param>
-              <param argument="cluster_field" type="text" value="" label="cluster_field" help="A string denoting the field in data, holding the cluster label for the data you want to correct">
+              <param argument="model" type="text" label="model" help="The model function to be fitted to the regimes. It must be a function of the form :math:`f(x, *p)`, where :math:`x` is the ``numpy.array`` holding the independent variables and :math:`p` are the model parameters that are to be obtained by fitting. Depending on the `x_date` parameter, independent variable x will either be the timestamps of every regime transformed to seconds from epoch, or it will be just seconds, counting the regimes length">
```

Comment:
The `CurveFitter` type is a function / Callable. Guess the SAQC method should be removed (no default values)